manifests/dev/vsphere-67u3/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -161,3 +165,23 @@ metadata:
 spec:
   attachRequired: true
   podInfoOnMount: false
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vsphere-csi-controller
+  namespace: kube-system
+  labels:
+    app: vsphere-csi-controller
+spec:
+  ports:
+    - name: ctlr
+      port: 2112
+      targetPort: 2112
+      protocol: TCP
+    - name: syncer
+      port: 2113
+      targetPort: 2113
+      protocol: TCP
+  selector:
+    app: vsphere-csi-controller
```

Comment:
Can you confirm if this yaml can be applied on a k8s cluster out of the box. I see a "no newline at end of file" warning.